### ISSUE TITLE
table_prefix becomes schema_prefix if a schema is defined

### ISSUE THREAD

**Issue by marcoreni** [DamienHarper/DoctrineAuditBundle#3]
Hello,

I'm currently mapping my entities using annotations, my database is PostgreSQL.

My entities are annotated as:
```
/**
 * @ORM\Entity
 * @ORM\Table(schema="my_schema", name="entity_name")
 */
```

I tried adding a table_prefix to my configuration, and `doctrine:schema:update --dump-sql` showed the following:
```
CREATE SCHEMA audit_my_schema;
     CREATE TABLE audit_my_schema.entity_name (id SERIAL NOT NULL, type VARCHAR(10) NOT NULL, object_id INT NOT NULL, diffs JSON DEFAULT NULL, blame_id INT DEFAULT NULL, blame_user VARCHAR(100) DEFAULT NULL, ip VARCHAR(45) DEFAULT NULL, created_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, PRIMARY KEY(id));
```

Is this expected behaviour? Is there a way to set a table_prefix while conserving the original schema?

**Comment by DamienHarper** [comment]
@marcoreni 
I've released 1.0.4 which fixes an issue with explicitly specifying schema name and using a table prefix.
In your case, it should produce 
`CREATE TABLE my_schema.audit_entity_name ...` 
instead of 
`CREATE TABLE audit_my_schema.entity_name ...`

Though, I haven't been able to reproduce the first part of your issue: `CREATE SCHEMA audit_my_schema;`

Can you please update to 1.0.4 and report me if it fixed your issue or not?

**Comment by marcoreni** [comment]
Thanks @DamienHarper , the new release fixed my issue:

```
$ sf doctrine:schema:update --dump-sql

 The following SQL statements will be executed:

     CREATE TABLE my_schema.audit_entity_name (id SERIAL NOT NULL, type VARCHAR(10) NOT NULL, object_id INT NOT NULL, diffs JSON DEFAULT NULL, blame_id INT DEFAULT NULL, blame_user VARCHAR(100) DEFAULT NULL, ip VARCHAR(45) DEFAULT NULL, created_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, PRIMARY KEY(id));
```

I think that Doctrine tried to create `audit_my_schema` because it didn't exist (I only created `my_schema`). Fixing the prefix issue solved this problem too.

Thanks again
